config/crd/bases/mcad.ibm.com_appwrappers.yaml

```diff
@@ -778,6 +776,10 @@ spec:
                   QueueJob (by Informer)
                 format: date-time
                 type: string
+              controllerfirstdispatchtimestamp:
+                description: Microsecond level timestamp when controller first dispatches appwrapper
```

Comment:
Could you make sure the description from the CRD and the documentation from the corresponding API field match?

Reply:
Done !